src/libstd/sys/unix/process/process_common.rs

```diff
@@ -199,6 +204,10 @@ impl Command {
     pub fn get_gid(&self) -> Option<gid_t> {
         self.gid
     }
+    #[allow(dead_code)]
+    pub fn get_groups(&self) -> &Option<Vec<gid_t>> {
```

Comment:
`&Option<Vec<gid_t>>` is not a useful type. It probably should be `Option<&[gid_t]>` for ease-of use. If we change to a `Vec<_>`, it could be `&[gid_t]`.

Reply:
I don't think this can be `Option<&[gid_t]>` (because it needs to own the slice), but it can be `Option<Box<[gid_t]>>`.

(It does need to be `Option`.)